sync/sync-impl/src/main/res/layout/fragment_sync_setup.xml

```diff
@@ -38,6 +38,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:padding="@dimen/keyline_4"
+            android:visibility="invisible"
```

Comment:
We have code in place for this button, but it's always invisible. Maybe we can remove the code?